### PR TITLE
feat: open space

### PR DESCRIPTION
**Pull request by Ritabear**
## Types of changes
- open the page of the open space
- open space i18n

## Description
- wait for more information to update pages
- After the other updates of the document, I will recheck the page.

## Related Issue
[#532](https://github.com/pycontw/pycontw-frontend/issues/532)

### CODE REVIEW

**Comment by SivanYeh** [comment]
文案上週有更新一版：https://docs.google.com/document/d/1WHCKUe2zYoG88bPHJfsvYm7doN7XcoeUkBuGDR5xt8w/edit

我把更新的部分標註下

File: i18n/events/open-spaces.i18n.js [hunk 1]
```diff
@@ -4,81 +4,87 @@ export default genI18nMessages({
     'en-us': {
         title: 'Open Space',
         intro:
-            'Open Spaces are self-organizing meetup events that happen simultaneously with the main conference. ' +
-            'Open Spaces are organized during the period of PyCon by attendees, ' +
-            'which provides a way for you to define, organize, ' +
+            'Open Spaces are self-organizing meetup events that occur simultaneously with the main conference. ' +
```
Comment (SivanYeh) [inline i18n/events/open-spaces.i18n.js:7]: -> "Open Space is a self-organizing meetup event that occurs simultaneously with the main conference."

File: i18n/events/open-spaces.i18n.js [hunk 2]
```diff
@@ -4,81 +4,87 @@ export default genI18nMessages({
...
+            'Organized by attendees during PyCon Taiwan, Open Spaces allow you to define, organize, ' +
```
Comment (SivanYeh) [inline i18n/events/open-spaces.i18n.js:8]: -> "Open Space allows you to define, organize"

File: i18n/events/open-spaces.i18n.js [hunk 3]
```diff
@@ -4,81 +4,87 @@ export default genI18nMessages({
...
+            'and plan meetups as you prefer.' +
             'plain out the meetup as you preferred. ' +
-            'Enjoy the Open Space while making new friends, ' +
-            'chat about any topic you’d like at the same time.{br}' +
-            'For those who have participated in previous PyCon Taiwan, ' +
-            'Open Spaces may sound familiar to you. ' +
-            'Yes! it’s basically the good old BoF (Birds of a Feather), ' +
-            'but with a name that is more comprehensive to new conference attendees.',
+            'Enjoy Open Spaces by making new friends and chatting about any topic you like.' +
+            'For those who have participated in past PyCon Taiwan or other conferences, ' +
```
Comment (SivanYeh) [inline i18n/events/open-spaces.i18n.js:12]: "For those who..."前加斷行

File: i18n/events/open-spaces.i18n.js [hunk 4]
```diff
@@ -5,112 +5,125 @@ export default genI18nMessages({
...
+                    'Each table will have a whiteboard displaying a specific topic.{br}{br}' +
+                    'Find a topic that interests you and use the memo to write down your thoughts and feedback.' +
+                    'If a topic is not listed yet, ' +
+                    'just write down what you would like to talk about on a non-topic board!',
             },
             time: {
                 title: 'Time',
-                description: 'Day 1: 12:05-17:10{br}Day 2: 12:05-17:10',
+                description:
+                    'Main Session: Day 2, 14:40–15:40' +
```
Comment (SivanYeh) [inline i18n/events/open-spaces.i18n.js:33]: 斷行-> "Main Session: Day 2, 14:40–15:40{br}"

File: i18n/events/open-spaces.i18n.js [hunk 5]
```diff
@@ -5,112 +5,125 @@ export default genI18nMessages({
...
+                    'Feel free to visit the service desk at any time to obtain a whiteboard, ' +
+                    'write down a topic, and initiate a discussion. ' +
+                    'Enjoy ‌lively and engaging conversations in the open space!',
             },
             subjects: {
                 title: 'Topics',
                 description:
                     'We don’t know either! It’s up to you and fellow attendees 😉{br}' +
```
Comment (SivanYeh) [inline i18n/events/open-spaces.i18n.js:41]: 句號-> 'We don’t know either! It’s up to you and fellow attendees. 😉{br}'

File: i18n/events/open-spaces.i18n.js [hunk 6]
```diff
@@ -5,112 +5,125 @@ export default genI18nMessages({
         title: 'Open Space',
         intro:
             'Open Spaces are self-organizing meetup events that happen simultaneously with the main conference. ' +
```
Comment (SivanYeh) [inline i18n/events/open-spaces.i18n.js:7]: -> 'Open Space is ...'

**Review by SivanYeh — CHANGES_REQUESTED** [review]
@Ritabear 最後一哩路了：
1. 把剩下4條我還沒刪除或resolve的comments 修改完成
2. 到codebase查詢任何複數型Open Spaces, 將它改成單數. 例如：NavBar.i18n.js裡面的Open Spaces
image

**Review by mattwang44 — APPROVED** [review]

**Pull request merged**